sway/commands/seat/shortcuts_inhibitor.c

```diff
@@ -0,0 +1,121 @@
+#include "sway/commands.h"
```

Comment:
Not a huge fan of this entire file. I'd recommend taking inspiration from https://github.com/swaywm/sway/blob/e1eda8a09c15780e7f6ab0b6a1ab7e811a3d8764/sway/commands/urgent.c

Something like:
```c
// Validate config->handler_context.seat_config is not NULL
if (strcmp(argv[0], "enable") == 0) {  // Possibly rename to `allow` for consistency
    sc->allow_shortcuts_inhibit = true;
} else if (strcmp(argv[0], "disable") == 0) {  // Possibly rename to `deny` for consistency
    sc->allow_shortcuts_inhibit = false;
} else if (strcmp(sc->name, "*") == 0) {
    return cmd_results(CMD_INVALID, "cannot activate/deactivate for wildcard seat");
} else if (!config->active) {
    return cmd_results(CMD_INVALID, "only enable and disable can be used in the config");
} else {
    // Validate seat exists
    // Validate inhibitor exists
    // Use parse_boolean(argv[0], inhibitor->active) to get state to set
}
```

Reply:
Done. Lost the operations enum. I stuck with enable/disable instead of allow/deny because deny in particular suggests that we have something to say in the matter, i.e. would actively refuse inhibitors. In think "disable" makes it more clear that we're just not going to honour the inhibitors. I pruned all other uses of "allow" to be consistent.